scripts/formatting.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+NC='\033[0m'
+
+function echo_error() {
+    echo -e "${RED}$1${NC}"
+}
+
+function echo_success() {
+    echo -e "${GREEN}$1${NC}"
+}
+
+function echo_warning() {
+    echo -e "${YELLOW}$1${NC}"
+}
```

Comment:
Could you please add comments for future developers? 
```suggestion
# Color definitions
RED='\033[0;31m'    # Red color for errors
GREEN='\033[0;32m'  # Green color for success messages
YELLOW='\033[0;33m' # Yellow color for warnings
NC='\033[0m'        # No Color (reset)

# Function to print error messages in red
function echo_error() {
    echo -e "${RED}$1${NC}"
}

# Function to print success messages in green
function echo_success() {
    echo -e "${GREEN}$1${NC}"
}

# Function to print warning messages in yellow
function echo_warning() {
    echo -e "${YELLOW}$1${NC}"
}
```

Reply:
> Could you please add comments for future developers?

Added comments.